sunpy/coordinates/utilities.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division
+import datetime
+import warnings
+
+import numpy as np
+import astropy.units as u
+from astropy.time import Time
+from astropy.coordinates import SkyCoord, Angle, Longitude, PrecessedGeocentric, AltAz, \
+                                get_body_barycentric
+from astropy.coordinates.representation import CartesianRepresentation, SphericalRepresentation
+from astropy._erfa.core import ErfaWarning
+
+from sunpy.time import parse_time
+
+from .frames import HeliographicStonyhurst as HGS
+from .transformations import _sun_detilt_matrix
+
+__all__ = ['get_earth', 'get_sun_B0', 'get_sun_L0', 'get_sun_P', 'get_sun_orientation']
+
+
+def get_earth(time='now'):
+    """
+    Return a SkyCoord for the location of the Earth at a specified time in the
+    HeliographicStonyhurst frame.  The longitude will be 0 by definition.
+
+    Parameters
+    ----------
+    time : various
+        Time to use in a parse_time-compatible format
+
+    Returns
+    -------
+    out : `~astropy.coordinates.SkyCoord`
+        SkyCoord for the location of the Earth in the HeliographicStonyhurst frame
+    """
+    obstime = Time(parse_time(time))
```

Comment:
they are converted to `datetime` objects (all output from `parse_time` is), so it would be better to catch them explicitly here to keep hold of times in other scales.